.buildkite/release.steps.yaml

```diff
@@ -88,3 +88,18 @@ steps:
     concurrency: 1
     concurrency_group: "unrealgdk-release"
     <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
+
+ # Stage 5: Mirror the release code from Github to Gitee
+  - block: "Unblock mirror code to gitee"
+    prompt: "This action will mirror all of the release code to gitee."
+
+
+  - trigger: platform-copybara
+    label: "Run platform-copybara"
+    depends_on: Release
```

Comment:
@mironec do you know if this should be `Release` or `"release"`?

I'm reading BuildKite's [defining explicit dependencies](https://buildkite.com/docs/pipelines/dependencies#defining-explicit-dependencies) documentation but, amusingly, it's not explicit about what `depends_on` accepts as input.

Reply:
I think this should work, curiosity of yaml, quotes are not necessary most of the time.